core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -189,7 +189,7 @@ object EntityDecoder {
 
   /** Helper method which simply gathers the body into a single Chunk */
   def collectBinary[F[_]: Concurrent](m: Media[F]): DecodeResult[F, Chunk[Byte]] =
-    DecodeResult.success(m.body.chunks.compile.toVector.map(bytes => Chunk.concat(bytes)))
+    DecodeResult.success(m.entity.body.chunks.compile.toVector.map(bytes => Chunk.concat(bytes)))
```

Comment:
Can you keep a `body` on `Message`?  A lot less would break.  It's `withBodyStream` that's now bad.

Reply:
body is back, but deprecated.